torch/csrc/jit/interpreter.cpp

```diff
@@ -1,42 +1,208 @@
+#include "Python.h"
 #include "interpreter.h"
 #include "torch/csrc/jit/ir.h"
 #include "torch/csrc/autograd/profiler.h"
 #include "torch/csrc/jit/generated/aten_dispatch.h"
+#include "Python.h"
+#include "pybind11/pybind11.h"
```

Comment:
I think it's a better idea to include `torch/csrc/jit/pybind.h`. It already defines some useful wrappers